quadlet/bluesky_pds/caddy.container

```diff
@@ -17,6 +17,7 @@ UserNS=auto:size=28
 
 [Service]
 Restart=on-failure
+ExecReload=/usr/bin/podman exec -w /etc/caddy --cidfile=%t/%N.cid caddy reload
```

Comment:
Instead of directly setting `ExecReload`, you can use Quadlet's `ReloadCmd` key: https://docs.podman.io/en/latest/markdown/podman-systemd.unit.5.html#reloadcmd